site/content/docs/5.3/examples/badges/index.html

```diff
@@ -82,16 +82,16 @@
 
 <div class="d-flex gap-2 justify-content-center py-5">
   <span class="badge d-flex p-2 align-items-center text-bg-primary rounded-pill">
-    <span class="px-1">Primary</span>
-    <a href="#"><svg class="bi ms-1" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
+    <span class="px-1">Primary 1</span>
+    <a href="#"><svg class="bi ms-1" width="16" height="16" role="img" aria-label="Delete primary 1"><use xlink:href="#x-circle-fill"/></svg></a>
```

Comment:
for links that contain an svg, I would say it might be cleaner to add the `aria-label` to the link, rather than the svg. (the svg doesn't then need any `role` or `aria-hidden`, because the `aria-label` on the link will override/erase anything that's actually inside it)

Reply:
doing it on the link rather than the svg then means the link would be announced cleanly as "Delete primary 1, link" rather than "Delete primary 1, graphic, link" (as the fact that this is a "graphic" is really not that important/relevant to the user, I'd argue.

Reply:
```suggestion
    <a href="#" aria-label="Delete primary 1"><svg class="bi ms-1" width="16" height="16"><use xlink:href="#x-circle-fill"/></svg></a>
```